src/query/wrap.js

```diff
@@ -0,0 +1,66 @@
+import Expr from '../Expr'
+import { Bytes } from '../values'
+import { checkInstanceHasProperty } from '../_util'
+import arity from './arity'
+import Lambda from './Lambda'
+
+/**
+ * Wraps an object as an Expression. This will automatically wrap any bare objects with
+ * the appropriate {@link object} escaping.
+ * @param {Object} obj
+ *  The object to be wrapped as an Expression.
+ * @returns {Expr}
+ *   The expression wrapping the provided object.
+ * @private
+ */
+export function wrap(obj) {
```

Comment:
why not put all those utility functions (like `wrap`, `varargs`, `params` and &c) into a single file?